age/smart-contract/assembly/contracts/main.ts

```diff
@@ -0,0 +1,78 @@
+// The entry file of your WebAssembly module.
+import { Args } from "@massalabs/as-types";
+import { Storage, generateEvent } from "@massalabs/massa-as-sdk";
+
+/**
+ * This function is the constructor, it is always called once on contract deployement.
+ * 
+ * In this case, the constructor initiate's the store by adding a 'person' entry with a default 'age' value.
+ * @remarks 
+ *  If the entry doesn't exist the person is created.
+ *  It also generates an event that indicates the changes that are made.
+ * 
+ * @param args The serialized arguments (unused).
+ * 
+ * @returns none
+ */
+
+export function constructor(_: StaticArray<u8>): StaticArray<u8> {
+    let age = new Args().add(1 as u32);
+    let name = new Args().add("alice");
+    
+    Storage.set(name.serialize(), age.serialize());
+    return [];
+}
+
+
+/**
+ * This functions changes the age of somone by a given name and a given new age value.
+ * 
+ * @remarks 
+ *  If the entry doesn't exist the person is created.
+ *  It also generates an event that indicates the changes that are made.
+ * 
+ * @param args The serialized arguments.
```

Comment:
Yous should explain the format. furthermore in tsdoc, @param tag is never followed by its type. The linter should have yell. Can you check why ?